src/server/protocol.ts

```diff
@@ -630,6 +631,32 @@ export interface GetMoveToRefactoringFileSuggestions extends Response {
     };
 }
 
+/**
+ * Request refactorings at a given position post pasting text from some other location.
+ */
+
+export interface GetPostPasteImportFixesRequest extends Request {
+    command: CommandTypes.GetPostPasteImportFixes;
+    arguments: GetPostPasteImportFixesRequestArgs;
+}
+export type DocumentRange = FileRangeRequestArgs;
+export type CopyRange  = {

```

Comment:
Can this be a `TextSpan`? Maybe also combine it with `orignalFile` and use a `FileSpan`